misc/json-to-record-converter/README.md

```diff
@@ -0,0 +1,530 @@
+
+<div>
+    <table>
+        <tbody>
+            <tr>
+                <td>Java Class</td>
+                <td>JSON Type</td>
+                <td>Ballerina Type&nbsp;</td>
+            </tr>
+            <tr>
+                <td>JsonElement</td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <li>JsonNull</li>
+                    </ul>
+                </td>
+                <td>Null</td>
+                <td>?/nil</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <li>JsonPrimitive</li>
+                    </ul>
+                </td>
+                <td>&nbsp;</td>
+                <td>&nbsp;</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <ul>
+                            <li>isBoolean</li>
+                        </ul>
+                    </ul>
+                </td>
+                <td>Boolean</td>
+                <td>boolean</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <ul>
+                            <li>isString</li>
+                        </ul>
+                    </ul>
+                </td>
+                <td>String</td>
+                <td>string</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <ul>
+                            <li>isNumber (contains &lsquo;.&rsquo;)</li>
+                        </ul>
+                    </ul>
+                </td>
+                <td>Number</td>
+                <td>decimal</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <ul>
+                            <li>isNumber</li>
+                        </ul>
+                    </ul>
+                </td>
+                <td>Integer</td>
+                <td>int</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <li>JsonObject</li>
+                    </ul>
+                </td>
+                <td>Object</td>
+                <td>record</td>
+            </tr>
+            <tr>
+                <td>
+                    <ul>
+                        <li>JsonArray</li>
+                    </ul>
+                </td>
+                <td>Array</td>
+                <td>array</td>
+            </tr>
+        </tbody>
+    </table>
+    <p>&nbsp;</p>
+</div>
+
+### Mapping of Null Type
+This section explains how different ways an optional Ballerina field can be generated.
+
+#### Scenario 01: JSON null
+When the value of a JSON field is null, it can be mapped as an optional field of [anydata](https://ballerina.io/spec/lang/master/#anydata) data type in ballerina.
```

Comment:
```suggestion
When the value of a JSON field is null, it can be mapped as a field of [anydata](https://ballerina.io/spec/lang/master/#anydata) data type in ballerina.
```